yt_dlp/extractor/franceculture.py

```diff
@@ -61,65 +31,68 @@ def _real_extract(self, url):
             'id': display_id,
             'title': self._html_search_regex(
                 r'(?s)<h1[^>]*itemprop="[^"]*name[^"]*"[^>]*>(.+?)</h1>',
-                webpage, 'title', default=self._og_search_title(webpage)),
+                webpage,
+                'title',
+                default=self._og_search_title(webpage),
+            ),
             'description': self._html_search_regex(
-                r'(?s)<div[^>]+class="excerpt"[^>]*>(.*?)</div>', webpage, 'description', default=None),
+                r'(?s)<meta name="description" content="([^"]+)',
+                webpage,
+                'description',
+                default=None,
+            ),
             'thumbnail': self._og_search_thumbnail(webpage),
             'uploader': self._html_search_regex(
-                r'(?s)<span class="author">(.*?)</span>', webpage, 'uploader', default=None),
-            'upload_date': unified_strdate(self._html_search_regex(
-                r'(?s)class="teaser-text-date".*?(\d{2}/\d{2}/\d{4})', webpage, 'date', default=None)),
+                r'(?s)<span class="author">(.*?)</span>',
+                webpage,
+                'uploader',
+                default=None,
+            ),
+            'upload_date': ''.join(
+                re.search(
+                    r'"datePublished":"(\d{4})-(\d{2})-(\d{2})T',  # type: ignore
+                    webpage,  # type: ignore
+                ).groups()
```

Comment:
> (\d{4})-(\d{2})-(\d{2})T

Isn't this ISO-8601 formatted date time? If so,

```suggestion
            'timestamp': unified_timestamp(
                self._search_regex(
                    r'"datePublished":"([^"]+?)"', 'timestamp', webpage, fatal=False))
```

Use `self._search_regex` everywhere unless you have to use `re.finditer` or `re.findall`

Reply:
The `unified_timestamp(` helper function doesn't help, but I switched to `self._search_regex(` ✅

Reply:
Implemented ✅